CHANGE_LOG

```diff
@@ -1,3 +1,227 @@
+Version 0.53.0 (04 February, 2021)
+----------------------------------
+
+This release continues to add new features, bug fixes and stability improvements
+to Numba.
+
+Highlights of core changes:
+
+* Support for Python 3.9 (Stuart Archibald).
+* Function sub-typing (Lucio Fernandez-Arjona).
+* Initial support for dynamic ``gufuncs`` (i.e. from ``@guvectorize``)
+  (Guilherme Leobas).
+* Parallel Accelerator (``@njit(parallel=True)`` now supports Fortran ordered
+  arrays (Todd A. Anderson and Siu Kwan Lam).
+
+Intel also kindly sponsored research and development that lead to two new
+features:
+
+  * Exposing LLVM compilation pass timings for diagnostic purposes (Siu Kwan
+    Lam).
+  * An event system for broadcasting compiler events (Siu Kwan Lam).
+
+Highlights of changes for the CUDA target:
+
+* CUDA 11.2 onwards (versions of the toolkit using NVVM IR 1.6 / LLVM IR 7.0.1)
+  are now supported (Graham Markall).
+* A fast cube root function is added (Michael Collison).
+* Support for atomic ``xor``, increment, decrement, exchange, are added, and
+  compare-and-swap is extended to support 64-bit integers (Michael Collison).
+* Addition of ``cuda.is_supported_version()`` to check if the CUDA runtime
+  version is supported (Graham Markall).
+* The CUDA dispatcher now shares infrastructure with the CPU dispatcher,
+  improving launch times for lazily-compiled kernels (Graham Markall).
+* The CUDA Array Interface is updated to version 3, with support for streams
+  added (Graham Markall).
+* Tuples and ``namedtuples`` can now be passed to kernels (Graham Markall).
+* Initial support for Cooperative Groups is added, with support for Grid Groups
+  and Grid Sync (Graham Markall).
```

Comment:
Just realised from reading the comprehensive list below that this should be:

```suggestion
  and Grid Sync (Graham Markall and Nick White).
```

Reply:
ah yes, well caught, didn't update both places.

Reply:
change accepted, thanks.